scripts/dockerify.py

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env python
+########
+# Copyright (c) 2016 GigaSpaces Technologies Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+#    * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    * See the License for the specific language governing permissions and
+#    * limitations under the License.
+
+from __future__ import print_function
+
+import argparse
+import os
+from functools import partial
+from subprocess import check_call, check_output, CalledProcessError
+from time import sleep
+
+import yaml
+
+
+EXPOSE = [22, 80, 443, 5671]
+
+# 22 isn't needed here as the user can access SSH via the internal address
+PUBLISH = ['80:80', '443:443', '5671:5671']
```

Comment:
Doesn't this (at least 5671) mean that the manager can't usefully deploy an agent to an external platform?